### PR TITLE
Added section about SystemJS

### PR DESCRIPTION
**Pull request by bennyn**
[SystemJS](https://github.com/systemjs/systemjs) plays a big role in modern JavaScript module management, thus I would like to provide a description on how to load **logdown.js** with SystemJS.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -24,42 +24,61 @@ or in the preview below.
...
-based on the prefix name, as we will see after.
+```js
+System.import('logdown').then(function(Logdown) {
+  var logger = new Logdown({prefix: 'foo'}
+});
+```
+
+### Other
+
+You can also use the lib in the browser in the same way as in the server if you use [Browserify](http://browserify.org/). Or you can just download it [here](https://github.com/caiogondim/logdown.js/archive/master.zip) and put the `dist/index.js` file in your public folder.
```
Comment (caiogondim) [inline readme.md]: I assume this is not related to the commit...?

The reason I put phrases in more than one line is that I don't like code and text to be bigger than 80 columns
Improves readability and avoids nesting

File: readme.md [hunk 2]
```diff
@@ -196,9 +215,7 @@ Logdown.disable('*foo*', '-foobar')
 
 ## Donating
 
-If you found this library useful and are willing to donate, transfer some
-bitcoins to `1BqqKiZA8Tq43CdukdBEwCdDD42jxuX9UY` or through the
-[URL](https://www.coinbase.com/caiogondim) https://www.coinbase.com/caiogondim
+If you found this library useful and are willing to donate, transfer some bitcoins to `1BqqKiZA8Tq43CdukdBEwCdDD42jxuX9UY` or through the [URL](https://www.coinbase.com/caiogondim) https://www.coinbase.com/caiogondim
```
Comment (caiogondim) [inline readme.md]: Same as I commented before
> I assume this is not related to the commit...?

> The reason I put phrases in more than one line is that I don't like code and text to be bigger than 80 columns
> Improves readability and avoids nesting

**Comment by bennyn** [comment]
@caiogondim I thought that keeping a width of 80 characters is a little outdated since this practice was invented when code was printed on paper. Keeping 80 characters width in Markdown leads to unnecessary text breaks as you can see here:

unbenannt

But to be consistent I reverted my change to these lines to stick to the rules. 😃

**Comment by caiogondim** [comment]
We could discuss over it for days

But it is nice to know you are passionate about software
Because I'm too 😄 

But I think that change is not inline with the commit message

**Comment by caiogondim** [comment]
@bennyn 
What is your npm username?
Will add you as owner of the package so you can make releases as well

**Comment by bennyn** [comment]
@caiogondim I am this guy: https://www.npmjs.com/~bennyn

**Comment by caiogondim** [comment]
Added you as owner
Let's work on your first publish on [that PR](https://github.com/caiogondim/logdown.js/pull/47)
